pkg/asset/manifests/imagedigestmirrorset.go

```diff
@@ -90,8 +91,8 @@ func ConvertImageDigestMirrorSet(imageDigestSources []types.ImageDigestSource) *
 		for mi, mirror := range imageDigestSource.Mirrors {
 			mirrors[mi] = apicfgv1.ImageMirror(mirror)
 		}
-
-		imageDigestMirrorSet.Spec.ImageDigestMirrors[idsi] = apicfgv1.ImageDigestMirrors{Source: imageDigestSource.Source, Mirrors: mirrors}
+		logrus.Debugf("SourcePolicy is %s for Source: %s", imageDigestSource.SourcePolicy, imageDigestSource.Source)
```

Comment:
```suggestion
		logrus.Debugf("SourcePolicy is %q for Source: %s", imageDigestSource.SourcePolicy, imageDigestSource.Source)
```

nit: if no source policy is defined, it will print:

```
SourcePolicy is for Source: source-name
```

Using `%q` will add a double quote around the empty value for easier read, WDYT?


```
SourcePolicy is "" for Source: source-name
```